apps/glusterfs/app.go

```diff
@@ -291,6 +291,11 @@ func (a *App) SetRoutes(router *mux.Router) error {
 			Method:      "POST",
 			Pattern:     "/devices/{id:[A-Fa-f0-9]+}/state",
 			HandlerFunc: a.DeviceSetState},
+		rest.Route{
```

Comment:
@raghavendra-talur Why would we use the device again if the brick data was moved?  Why move it then?